sklearn/cluster/_dbscan.py

```diff
@@ -86,7 +86,9 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski', metric_params=None,
         The number of parallel jobs to run for neighbors search.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
-        for more details.
+        for more details. Parallel execution is implemented for computing the 
+        distances. If precomputed distance are used, parallel execution is not 
```

Comment:
Thanks you. Probably better to  remove "Parallel execution is implemented for computing the distances." That's redundant with the first sentence in this paragraph. Also "If precomputed distance is used"? Otherwise LGTM.

Reply:
Thanks wil remove the redundancy. The ticket is about parallel execution not making a difference when precomputed distances are used and clarifying that in the documentation.  Thats way "If precomputed distance are used" is in there but should perhaps be updated?

Reply:
Yes, sounds good.